cypress/integration/DynamicAttributes/IdSchemeWithCommonAttributes/index.js

```diff
@@ -17,24 +15,24 @@ Given(
 
         if (left === 'org units') {
             cy.route({
-                url: `${loginUrl}/api/attributes.json?paging=false&fields=id,displayName&filter=unique:eq:true&filter=organisationUnitAttribute:eq:true`,
+                url: /api\/\d{2}\/attributes\?filter=unique:eq:true&filter=organisationUnitAttribute:eq:true&fields=id,displayName&paging=false/,
```

Comment:
This (and the following changes) break tests against the `debug` backend as the root url contains a path `/dev` (contrary to `/` when using localhost). This will def. be the case when doing proper e2e tests in the future, so this should def. be considered, I guess just adding back the `loginUrl` should suffice.

Reply:
This (and all the other requests) are happening like this:

```
filter=unique:eq:true,organisationUnitAttribute:eq:true
```

I don't know which version is correct, but the test doesn't match the request.

Reply:
I think the `app-shell` causes this issue.
It still depends on an older version of `app-runtime`.

Removing the `.d2` and `node_modules` folder and reinstalling the dependencies makes all the tests pass. I think we can wait for the `app-shell`s dependencies to be updated before merging this branch.

Reply:
@awgaan you shouldn't need to upgrade the dependency in `app-shell`, just upgrade it in the app's dependency tree - `yarn upgrade @dhis2/app-runtime`

Reply:
If there's a local `app-runtime` dep it means you have two copies which also won't work, but you can run `npx yarn-deduplicate --packages @dhis2/app-runtime` as well if needed

Reply:
There's only one version of `app-runtime` in your `yarn.lock` and it's the latest, so the `app-runtime` version isn't the issue here https://github.com/dhis2/import-export-app/blob/081146381f99f6a81939b8fd981dab516a227db2/yarn.lock#L1317